sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/FlowAnalysisExtensions.cs

```diff
@@ -23,14 +23,16 @@
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Diagnostics;
-using SonarAnalyzer.Helpers.FlowAnalysis.Common;
+using SonarAnalyzer.Helpers;
+using SonarAnalyzer.SymbolicExecution.CFG;

```

Comment:
Just random thought, if the base class is prefixed with `Base` or `Abstract` you won't have collision with the namespace.